src/topics/unread.js

```diff
@@ -135,7 +135,8 @@ module.exports = function (Topics) {
 		});
 
 		tids = await privileges.topics.filterTids('topics:read', tids, params.uid);
-		const topicData = (await Topics.getTopicsFields(tids, ['tid', 'cid', 'uid', 'postcount', 'deleted'])).filter(t => !t.deleted);
+		const topicData = (await Topics.getTopicsFields(tids, ['tid', 'cid', 'uid', 'postcount', 'deleted', 'scheduled']))
+			.filter(t => t.scheduled || !t.deleted);
```

Comment:
Shouldn't this be `!t.scheduled && !t.deleted`? We don't want scheduled and deleted to appear in unread topics.

Reply:
Here we hide "deleted topics", but privileged users' unread count is updated as usual for scheduled topics (combined with the filterTids a line above). For the unprivileged, theirs will be increased when the topic is published.